r-package/grf/tests/testthat/test_causal_forest.R

```diff
@@ -250,6 +250,42 @@ test_that("Weighting is roughly equivalent to replication of samples", {
   expect_true(mean(predict(causal.forest.rep, X[test, ]) > 100 + predict(causal.forest.biased, X[test, ])) >= .5)
 })
 
+test_that("sample weighting is practically identical to replicating samples", {
```

Comment:
Does this test give additional confidence over the one directly above? If not, we could drop it.

Reply:
It does not, I removed it